webapi/src/it/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala

```diff
@@ -847,12 +839,7 @@ class KnoraSipiIntegrationV1ITSpec
         baseApiUrl + "/v1/resources/" + URLEncoder.encode(pdfResourceIri.get, "UTF-8")
       ) ~> addCredentials(BasicHttpCredentials(userEmail, password))
 
-      val documentResourceResponse: JsObject = getResponseJson(documentResourceRequest)
-      val locdata                            = documentResourceResponse.fields("resinfo").asJsObject.fields("locdata").asJsObject
-      val nx                                 = locdata.fields("nx").asInstanceOf[JsNumber].value.toInt
-      val ny                                 = locdata.fields("ny").asInstanceOf[JsNumber].value.toInt
-      assert(nx == testPdfWidth)
-      assert(ny == testPdfHeight)
+      getResponseJson(documentResourceRequest)
```

Comment:
I don't understand what this test is actually testing. Shouldn't there at least be something to assert, e.g. the new filename?

Reply:
Good point, I'm not sure myself.